src/Farmer/Arm/DocumentDb.fs

```diff
@@ -215,6 +216,16 @@ type DatabaseAccount =
                             | [], Disabled -> null
                             | locations, _ -> box locations
                         publicNetworkAccess = string this.PublicNetworkAccess
+                        ipRules = 
+                          match this.RestrictToAzureServices with
+                          | Enabled ->
+                                box
+                                    [
+                                        {|
+                                            ipAddressOrRange = "0.0.0.0"
```

Comment:
I would have thought this allowed any IP?

Reply:
I thought that too, but not according to the [docs](https://learn.microsoft.com/en-us/azure/cosmos-db/how-to-configure-firewall#allow-requests-from-global-azure-datacenters-or-other-sources-within-azure). I've tested it and it does work, I have a cosmos DB with this deployed and I can only access it from an Azure VM.